crates/uv-cli/src/lib.rs

```diff
@@ -3174,6 +3174,11 @@ pub enum ToolCommand {
     ///
     /// Packages are installed into an ephemeral virtual environment in the uv
     /// cache directory.
+    #[command(
+        about = "Run a command provided by a Python package. Also available via the alias `uvx`.",
+        after_help = "You can also use `uvx` as an alias for `uv tool run`. \n\
```

Comment:
Oh there's already a comment for this above in the long help i.e. `uv help tool run`. I'm a bit hesitant to add it to the short help?

Reply:
Sorry! Do you mean `Run a command provided by a Python package` or something else entirely? 😬

Reply:
Hoping i understood that you meant the former: [73d1b39](https://github.com/astral-sh/uv/pull/7695/commits/73d1b393f9b54e46446206afdf73ecbfefa97fbb)
```
➜ ./target/debug/uv tool run --help | head
Also available via the alias `uvx`.

Usage: uv tool run [OPTIONS] [COMMAND]

...
```

Reply:
`uv help tool run` already includes 

> `uvx` is provided as a convenient alias for `uv tool run`, their behavior is identical.

So we'd have a duplicate message here which feels weird.

Reply:
I think if anything we'd use just `after_help` and not modify the `about`? You'd need to set `after_long_help` to an empty string.

Reply:
Maybe [882b4d3](https://github.com/astral-sh/uv/pull/7695/commits/882b4d395d1d905fa19c5851f7b46260605217d8)?